src/main/resources/standardTestCases/fofn_caching.test

```diff
@@ -0,0 +1,13 @@
+name: fofn_caching
+testFormat: runtwiceexpectingcallcaching
+backends: [Jes-Caching-No-Copy]
+
+files {
+  wdl: fofn_caching/fofn_caching.wdl
+}
+
+metadata {
+  workflowName: fofn_caching
+  status: Succeeded
+  "calls.fofn_caching.use_fofn.callCaching.result": "Cache Hit: <<CACHE_HIT_UUID>>:fofn_caching.use_fofn:-1"
```

Comment:
Because the input to this task (which is a fofn), will have the same content, hence hash.
That is because the shards of the scatter, when being call cached, output the original file path (of the call they're caching to). This results in the same file being written by `write_lines`, which will make this call cache hit.